dev/ci/ci-common.sh

```diff
@@ -32,7 +32,7 @@ then
     # Full Dune build, we basically do what `dune exec --` does
     export OCAMLPATH="$PWD/_build/install/default/lib/$OCAMLFINDSEP$OCAMLPATH"
     export COQBIN="$PWD/_build/install/default/bin"
-    export COQLIB="$PWD/_build/install/default/lib/coq"
+    export COQLIB="$PWD/_build/install/default/lib/rocq"
```

Comment:
NB this change is only for local running of the ci scripts, the actual CI running on gitlab uses the other branch of `if [ -n "$GITLAB_CI" ]` which doesn't set COQLIB.